crates/proof-of-sql/src/sql/ast/aggregate_expr.rs

```diff
@@ -0,0 +1,78 @@
+use super::ProvableExprPlan;
+use crate::base::commitment::Commitment;
+use proof_of_sql_parser::intermediate_ast::AggregationOperator;
+
+use super::{ProvableExpr, ProvableExprPlan};
+use crate::{
+    base::{
+        commitment::Commitment,
+        database::{Column, ColumnRef, ColumnType, CommitmentAccessor, DataAccessor},
+        proof::ProofError,
+    },
+    sql::proof::{CountBuilder, ProofBuilder, VerificationBuilder},
+};
+use bumpalo::Bump;
+use serde::{Deserialize, Serialize};
+use std::collections::HashSet;
+
+/// Provable aggregate expression
+///
+/// Currently it doesn't do much since aggregation logic is implemented elsewhere
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct AggregateExpr<C: Commitment> {
```

Comment:
Might be nice to name this to indicate that it doesn't actually do the aggregation. I don't really like doing `...able` and I can't think of anything better, so maybe just keep it as it is.
```suggestion
pub struct AggregatableExpr<C: Commitment> {
```

Reply:
@JayWhite2357 Datafusion names it [AggregateFunction](https://docs.rs/datafusion/latest/datafusion/logical_expr/expr/struct.AggregateFunction.html) which is also a bit confusing in its own way.

Reply:
Up to you what you want to name it.